src/board/dom-helpers.mjs

```diff
@@ -0,0 +1,15 @@
+function removeElementByClass(className) {
```

Comment:
i really love that you separated this into a new file! i was a little confused about what was in here at first because of the file name. At Sparkbox we usually name the file or folder functions like this go in `collaborators` or  just `helpers`

Reply:
Do you have any suggestions for naming this since I already have a `helpers.mjs` file?

Reply:
hmm yeah i think there are a couple of options:
- you could create a helpers folder and move what is currently in the `helpers.mjs` file into that folder with a new name, and move this into that folder with `dom.mjs`
- you could move these methods into the already existing `helpers.mjs` file

if that is really going to mess things up, i would say it is probably not worth it though and could totally stay as is!